code/edgemedia/src/main/java/com/adobe/marketing/mobile/edge/media/internal/MediaCollectionTracker.java

```diff
@@ -23,7 +23,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-class MediaCollectionTracker implements MediaTrackerInterface {
+class MediaCollectionTracker implements MediaEventTracking {
```

Comment:
Let's rename this class to `MediaEventTracker ` similar to iOS

Reply:
I'll leave that for future work in another PR when we add the MediaSession and MediaEventTracker classes.